dali/operators/generic/erase/erase.cc

```diff
@@ -27,106 +27,132 @@
 namespace dali {
 
 DALI_SCHEMA(Erase)
-  .DocStr(R"code(Erases one or multiple regions from the image.
+  .DocStr(R"code(Erases one or more regions from the image.
 
 The region is specified by an *anchor* (starting point) and a *shape* (dimensions).
 Only the relevant dimensions are specified.
-Non-specified dimensions are treated as if the whole range of the axis was provided.
+Unspecified dimensions are treated as if the entire range of the axis was provided.
```

Comment:
```suggestion
Not specified dimensions are treated as if the entire range of the axis was provided.
```
https://wikidiff.com/unspecified/nonspecified

Reply:
Done